fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java

```diff
@@ -580,6 +584,16 @@ public static RemoteFileInputFormat toRemoteFileInputFormat(String inputFormat)
         return RemoteFileInputFormat.fromHdfsInputFormatClass(inputFormat);
     }
 
+    public static RemoteFileInputFormat toRemoteFileInputFormat(String inputFormat, String serializationLib) {
```

Comment:
we can just have one method `toRemoteFileInputFormat`

Reply:
Ok, I will move the sd check logic to the upper layer